packages/mdc-menu/simple/foundation.js

```diff
@@ -150,109 +168,27 @@ class MDCSimpleMenuFoundation extends MDCFoundation {
   }
 
   /**
-   * Calculates transition delays for individual menu items, so that they fade in one at a time.
-   * @private
+   * @param {!Corner} corner Default anchor corner alignment of top-left menu corner.
```

Comment:
This isn't always controlling the top-left menu corner, right? Doesn't it depend on anchor position within viewport?

Reply:
I think we generally need to be more clear that the corner clients set is the "ideal" corner. But there is internal logic which will change the "actual" corner depending on the view port.